### PR TITLE
feat: Add jest-configurator to Watermark package

### PR DESCRIPTION
**Pull request by jimhunty**
As the title suggests, this applies Jest Configurator to the Watermark package.

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/fbf4316f34268bbb6bdb7cbbd533c3f738d26799/dextrose-test/index.html

File: packages/watermark/package.json [hunk 1]
```diff
@@ -4,24 +4,16 @@
   "description": "Watermark",
   "main": "watermark",
   "scripts": {
-    "test": "jest --bail --ci --coverage",
+    "test": "yarn test:android -- --ci --bail --coverage && yarn test:ios -- --ci --bail --coverage && yarn test:web -- --ci --bail --coverage",
```
Comment (colincclark) [inline packages/watermark/package.json]: `yarn` no longer needs the `-- --ci` double dashes

File: packages/watermark/package.json [hunk 2]
```diff
@@ -52,6 +44,7 @@
     "react-test-renderer": "16.2.0"
   },
   "dependencies": {
+    "@times-components/jest-configurator": "0.0.23",
```
Comment (colincclark) [inline packages/watermark/package.json]: `devDependencies`

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/e6e682a097e548d5b4f676ec889ee56888424bbb/dextrose-test/index.html

**Pull request merged**